includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -1576,10 +1576,12 @@ public function wc_setup_activate_save() {
 			exit;
 		}
 
-		$redirect_url   = site_url( add_query_arg( array(
+		$redirect_url   = add_query_arg( array(
+			'page'           => 'wc-setup',
+			'step'           => 'activate',
 			'from'           => 'wpcom',
 			'activate_error' => false,
-		) ) );
+		), get_admin_url() );
```

Comment:
The original redirect (and the rest of the codebase) uses `admin_url`:
https://github.com/woocommerce/woocommerce/blob/e296a276e3c540188d77e5494f7eba15209c7f52/includes/admin/class-wc-admin.php#L150 Probably a trivial concern but FYI in case it makes any difference.